src/components/Frame/components/ContextualSaveBar/components/DiscardConfirmationModal/tests/DiscardConfirmationModal.test.tsx

```diff
@@ -7,6 +7,22 @@ import {Modal} from 'components';
 import DiscardConfirmationModal from '../DiscardConfirmationModal';
 
 describe('<DiscardConfirmationModal />', () => {
+  it('passes its open prop value to the Modal', () => {
+    const discardConfirmationModalOpen = shallowWithAppProvider(
+      <DiscardConfirmationModal open onDiscard={noop} onCancel={noop} />,
+    );
+    expect(discardConfirmationModalOpen.find(Modal).prop('open')).toBe(true);
+
+    const discardConfirmationModalClosed = shallowWithAppProvider(
```

Comment:
Do we need the second mount in this test?

```suggestion
    const discardConfirmationModalClosed = mountWithAppProvider(
```